components/Avatar/avatar.css

```diff
@@ -0,0 +1,74 @@
+.avatars-wrapper {
+    background-color:var(--white);
+    display: flex;
+    align-items: baseline;
+    justify-content: center;
+    justify-self: center;
+    align-self: center;
+    padding: 1rem;
+    width: 80%;
+    margin: 1rem;
+
+}
+
+.avatar {
+    object-fit: cover;
+    height: 5rem;
+    width: 5rem;
+    margin: 1rem;
+
+}
+
+.avatar-img {
+    width: 100%;
+    height: 100%;
+}
+
+.avatar, .avatar-img {
+    border-radius:50%;
+}
+
+.avatar-xs {
+    height:3.5rem;
+    width:3.5rem;
+}
+
+.avatar-sm {
+    height: 5rem;
+    width: 5rem;
+}
+
+.avatar-md {
+    height: 6.5rem;
+    width: 6.5rem;
+}
+
+.avatar-lg {
+    height: 8rem;
+    width: 8rem;
+}
+
+.avatar-text {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color:var(--primary-color);
+    color: var(--secondary-color);
+    font-weight: 600;
+}
+
+.avatar.avatar.xs.avatar-text {
+    font-size: 1.75rem;
+}
```

Comment:
I think here you should avoid the too much nesting of class. This can mess up the `specificity`. Also you will not be able to use some general properties to all elements for ex. `color:red;` .

Reply:
Thank you for the review. That also helped me see the mistake in the code. The second class was supposed to be `avatar-xs` but I had typed `avatar.xs` which was not giving the required styling. Also removed the nested `avatar` class as I realized that was unnecessary. Thank you for the review.